contrail.sh

```diff
@@ -76,7 +76,7 @@ LAUNCHPAD_REPO=${LAUNCHPAD_BRANCH:-snapshots}
 
 USE_DISCOVERY=${USE_DISCOVERY:-False}
 
-RABBIT_USER=${RABBIT_USER:-guest}
+RABBIT_USER=${RABBIT_USER:-rabbitcontrail}
```

Comment:
Would it make sense to preserve the default of "guest"  ? It would still work ok on single node systems.